### PR TITLE
Crkbd/keymaps/oo

### PR DESCRIPTION
**Pull request by oliverocean**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by oliverocean** [comment]
Hi everyone! New contributor here, loving QMK on my newly soldered/assembled crkbd keeb.

I've tested my keymap (using it currently) and all works well; please let me know if my PR is setup correctly and/or if any changes are needed.

Thanks! :grin:

File: keyboards/crkbd/keymaps/oo/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+# Build instructions
```
Comment (fauxpark) [inline keyboards/crkbd/keymaps/oo/README.md:2]:
```suggestion
# Build instructions
```
And could you please rename this file to lowercase?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/crkbd/keymaps/oo/config.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
```
Comment (drashna) [inline keyboards/crkbd/keymaps/oo/config.h:27]: Shouldn't actually be needed, as this is the default behavior.

File: keyboards/crkbd/keymaps/oo/config.h [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#define USE_SERIAL_PD2
```
Comment (drashna) [inline keyboards/crkbd/keymaps/oo/config.h:31]: I don't believe that this is actually needed.
```suggestion
```

File: keyboards/crkbd/keymaps/oo/README.md (hunk 1 above)
Comment (oliverocean) [reply keyboards/crkbd/keymaps/oo/README.md]: Done and done, @fauxpark!

File: keyboards/crkbd/keymaps/oo/config.h (hunk 2 above)
Comment (oliverocean) [reply keyboards/crkbd/keymaps/oo/config.h]: Good to know and removed, thanks @drashna!

File: keyboards/crkbd/keymaps/oo/config.h (hunk 3 above)
Comment (oliverocean) [reply keyboards/crkbd/keymaps/oo/config.h]: Removed this as well and other commented out code so the file only has what is used.

**Comment by oliverocean** [comment]
Thanks for the quick feedback, @drashna and @fauxpark. I made all the requested changes.

At first my new build didn't compile :thinking: But then I ran ```qmk setup``` again and voila it worked! I forgot that I'd re-cloned the repo and some of the libraries were missing. (I put a step in the readme for others or more likely my future self). The CLI tools for this project are just lovely.

So this keymap is tested and I'm using it right now.

File: keyboards/crkbd/keymaps/oo/config.h [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define SPLIT_USB_DETECT
```
Comment (drashna) [inline keyboards/crkbd/keymaps/oo/config.h:23]: Is this required for your board?

Comment (oliverocean) [reply keyboards/crkbd/keymaps/oo/config.h:23]: Hi @drashna, this is a good question. I remember trying a few configurations earlier in the process of setting up this board but I couldn't remember if I'd specifically tested all the options so I reviewed the docs in the [Split Keyboard](https://docs.qmk.fm/#/feature_split_keyboard?id=hardware-configuration-options) section.

It indicates that the rules.mk file can contain ```SPLIT_KEYBOARD = yes``` or the config.h can contain ```#define SPLIT_USB_DETECT```. Further down the page it does indicate that the config.h define may also be optional for Pro Micro boards (which I am using for both board halves).

I went ahead and tested all the scenarios. Using neither option or just the rules.mk statement both result in the right side board thinking it is a master (instead of a slave) even when the usb is plugged into the left. But when I have the define in config.h, that does result in the right side board properly operating as a slave to the left side.

tl;dr, yes, this is required for my board.

**Review by drashna — APPROVED** [review]

**Comment by oliverocean** [comment]
@fauxpark did you have any other requested changes or does it look ok?

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by oliverocean** [comment]
Yeah! :partying_face:
